.drone.yml

```diff
@@ -1327,13 +1327,10 @@ steps:
       # increment these variables when a new major/minor version is released to bump the automatic builds
       # this only needs to be done on the master branch, as that's the branch that the Drone cron is configured for
       # build major version images which are just teleport:x
-      CURRENT_VERSION_ROOT: v5
-      # normally we only support the current release plus the two previous releases, but we're temporarily adding an
-      # extra version here during the migration over from x.y -> x.y-1 support to x -> x-1 support as per RFD 12.
-      OLD_CURRENT_VERSION_ROOT: v5.1
-      OLD_PREVIOUS_VERSION_ONE_ROOT: v5.0
-      OLD_PREVIOUS_VERSION_TWO_ROOT: v4.4
-      OLD_PREVIOUS_VERSION_THREE_ROOT: v4.3
+      CURRENT_VERSION_ROOT: v6
+      OLD_CURRENT_VERSION_ROOT: v5
```

Comment:
should this be `PREVIOUS_VERSION_ONE_ROOT`?
and should we remove v4.3?

Reply:
Yes to both!